book/src/integrations.md

```diff
@@ -22,7 +22,7 @@ from the Askama test suite for more on how to integrate.
 
 ## Axum integration
 
-Enabling the `with-axum` feature appends an implementation of Axum's
+Enabling the `with-axum` feature and depending on the `askama_axum` crate appends an implementation of Axum's
```

Comment:
If you depend on askama_axum I don't think you have specifically enable the `with-axum` flag, have you tried that?

Reply:
If the flag is not there, it doesn't seem to use the askama_axum crate at all. Whatever axum specific integration you have done with the askama_axum crate isn't seemingly touched without that flag

Reply:
Hello,
I just tried it. If I remove the `with-axum` flag, then having the `askama_axum` crate seemingly makes no difference?

Reply:
Yea, can confirm that. 

If you have the `with-axum`  feature, you need to have that dependency. However,  askama seems to works with axum even without the `askama_axum` crate? 

I am confused!

Reply:
I'm sorry, I don't have much time to dig into this. The feature flag is forward to the askama_derive crate, which in its `generator` generates some source code specific to the Axum integration. For many of the integrations, the generated source code depends on the integration crate but that may not be true here? Please have a look at the relevant source code yourself.

Reply:
Sure thing, I'll use that information to the best of my abilities.

Reply:
I can confirm that when the `with-axum` flag is used, the generated source code depends on the `askama_axum` integration crate. 

I can also confirm, that without the `with-axum` feature flag, the integration code is still generated. I can call the `into_reponse` function that is defined in the `askama_axum` on a Template.